System/Cryptography/Encryption.py

```diff
@@ -1,25 +1,37 @@
 
-def gen_secret(privatekey,publickey):
-    secret = pow(int(publickey), int(privatekey), prime)
-    
-    secret_bytes = str(secret)
+def GenerateSecret(PrivateKey, PublicKey): # Generates A New Secret #
+    Secret = pow(int(PublicKey), int(PrivateKey), Prime)
+    SecretBytes = str(Secret)
 	
-    #Generate hash key using SHA256
-    key = SHA256.new()
-    key.update(bytes(secret_bytes))
-    secretKey = key.hexdigest()
-    return secretKey
-
-def encrypt(Logger:object, Command:str):
-    PublicKey,PrivateKey= GenKeys(Logger)
-    SecretKey= gen_secret(PublicKey,PrivateKey)
+    # Generate Hash Key Using SHA256 #
+    Key = SHA256.new()
+    Key.update(bytes(SecretBytes))
+    SecretKey = Key.hexdigest()
+    return SecretKey
+
+def Encrypt(Logger:object, Command:str):
+    
+    # Load The Pub/Private Keys #
+    PublicKey, PrivateKey = ReadKeys(Logger)
+    SecretKey = GenerateSecret(PublicKey, PrivateKey)
     enc_Command= ""

```

Comment:
@PRISHIta123 Is this the text being actually encrypted? If so: firstly, where's the decryption function, and additionally, would it be better to use a module for this?

It may also be good to move this into a class so it's not using global vars.

Reply:
The commands in the CLI are being encrypted, it is being sent as a parameter to the Encrypt function.
If you would like me to add a Decryption function, I can do that as well, as I was only working on the encryption part for now.

Reply:
Yeah, it'd probably be best if we also added a decryption function so we can validate if this works like we think it should.

Reply:
I have added that now and renamed the Encryption file